lib/fog/kubevirt/compute/models/persistentvolume.rb

```diff
@@ -8,17 +8,17 @@ class Persistentvolume < Fog::Model
         attribute :uid,              :aliases => 'metadata_uid'
         attribute :annotations,      :aliases => 'metadata_annotations'
         attribute :labels,           :aliases => 'metadata_labels'
-		attribute :access_modes,	 :aliases => 'spec_access_modes'
-		attribute :mount_options,	 :aliases => 'spec_mount_options'
-		attribute :reclaim_policy,   :aliases => 'spec_reclaim_policy'
-		attribute :storage_class,    :aliases => 'spec_storage_class'
+        attribute :access_modes,     :aliases => 'spec_access_modes'
+        attribute :mount_options,    :aliases => 'spec_mount_options'
+        attribute :reclaim_policy,   :aliases => 'spec_reclaim_policy'
+        attribute :storage_class,    :aliases => 'spec_storage_class'
         attribute :capacity,         :aliases => 'spec_capacity'
         attribute :claim_ref,        :aliases => 'spec_claim_ref'
-        attribute :type,			 :aliases => 'spec_type'
-        attribute :config,			 :aliases => 'spec_config'
+        attribute :type,	     :aliases => 'spec_type'
+        attribute :config,	     :aliases => 'spec_config'
```

Comment:
these 2 lines need 1 space before `:aliases` to be alidned

Reply:
My intention was to align them. It seems that I missed some tabs